operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -1163,6 +1167,33 @@ def _writeIstioNAP(self, name, server, listen_address, listen_port, protocol, ht
     self.undent()
     self.writeln('</d:network-access-point>')
 
+  def getCoherenceClusterSystemResourceOrNone(self, serverOrTemplate):
```

Comment:
I don't think this is the right test.  We should add/replace if  the domain is a managed coherence cluster.  That is having a 

resources:
       CoherenceClusterSystemResource:

defined in the WDT model or at the domain level in WLST mean,   then we are going to add/replace the unicast address.  i.e. the user may not have the CoherenceClusterSystemResource in the Server/ServerTemplate level.   Also, you need to check whether the root level CoherenceClusterSystemResource is targeted to the cluster and only do it for those are targeted.